.github/template-bugReport.md

```diff
@@ -0,0 +1,18 @@
+**Issue Title:** Bug: [brief description of bug]
+
+Copy-and-paste the following template to the issue tracker and fill it in with relevant details:
+```markdown
+**The environment in which the bug is encountered (e.g. live server at `V5.93`, `master` branch at commit 1234567)**
+
+
+
+**Steps to reproduce the bug**
```

Comment:
Why is it for consistency reason? There is no `:` anywhere in all template headings.